test/Microsoft.ML.CpuMath.PerformanceTests/Microsoft.ML.CpuMath.PerformanceTests.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.2</LangVersion>
+    <StartupObject>Microsoft.ML.CpuMath.PerformanceTests.Program</StartupObject>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- The SDK we currently restore and use has 2.0 as the maximum supported version -->
+    <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <Compile Remove="BenchmarkDotNet.Artifacts\**" />
```

Comment:
I wonder if there is a property we can set to put these artifacts into the top-level `bin` folder instead.

@tannergooding - do you know?  I'll ping Adam, if not.

Reply:
I know you can, by setting `IConfig.ArtifactsPath` (programmatically), but I don't remember how to do that from MSBuild.

Reply:
I think you can leave it for now.  We have the same in the Benchmark tests that are already checked in:  https://github.com/dotnet/machinelearning/blob/697ea3d73d1dd68d516f46b45124ccd6ebe063f2/test/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks.csproj#L8-L12